rules/0340-puppet_rules.xml

```diff
@@ -53,7 +53,7 @@
 
     <!--
     Permission
-    - Oct  1 17:56:52 puppet puppet-master[7879]: Permission denied - /etc/puppet/modules/ossec/files/ossec-agent-win32-2.8.exe
+    - Oct  1 17:56:52 puppet puppet-master[7879]: Permission denied - /etc/puppet/modules/wazuh/wazuh-agent-4.1.0-1.msi
```

Comment:
The _files_ word is removed from the path. Was it intentional ?

Reply:
Yes. The path where the installer will be placed is customizable, but by default, Wazuh Puppet module places it there.

Reply:
Excellent!